### DISCUSSION TITLE
xdrip stopped uploading to nightscout

### DISCUSSION THREAD

**Discussion by pweigmann** [NightscoutFoundation/xDrip#2836] [Q&A] [answered]
For no apparent reason, xdrip stopped uploading to my railway hosted nightscout app last night. Updated to the latest nightly build which didn't help, nightscout is up to date as well. Error message reads: "Unable to do REST API Upload: marking record: failed" Any ideas? Missed 240 readings so far.

**Comment by Navid200** [comment, marked as answer]
We have changed nothing in xDrip related to Nightscout for a very long time.

Are you sure your Nightscout database is not full?

**Reply by pweigmann** [reply]
It's at 35%. Hmm, could also be a railway problem then?

**Reply by Navid200** [reply]
If you look at your xDrip Nightscout settings, do they agree with the following, specially the api/v1 at the end of the base URL?
https://navid200.github.io/xDrip/docs/Nightscout-Settings.html

**Reply by pweigmann** [reply]
Actually, I have to correct myself. Deploy log says "MongoError: you are over your space quota, using 512 MB of 512 MB", for some reason nightscout shows 175/512 MB are used. Thanks for the help!

**Reply by pweigmann** [reply]
In case anybody else has this problem - in my case Atlas loaded some sample collections into my databases, clogging it up. Deleting them solves the problem.
